change/@fluentui-react-conformance-14912322-4e02-4c42-9302-bd48ffd81217.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
```

Comment:
As you renaming `consistent-callback-args`, it should be a major bump (as it's a breaking change).

Reply:
react-conformance is on version 0.x.x. I think minor here is for breaking change.